changelog/add_local_to_rails_unknown_env.md

```diff
@@ -0,0 +1 @@
+* [#893](https://github.com/rubocop/rubocop-rails/issues/893): Support `local` as an environment for `Rails/UnknownEnv` from Rails 7.1 onward. ([@ghiculescu][])
```

Comment:
Does this warrant updating the cop's `VersionChanged` config?

Reply:
Probably? I'm not very familiar with how this works. Do I add the next version number or is there some other workflow for this?